recipes/cpr/all/conanfile.py

```diff
@@ -18,16 +22,14 @@ class CprConan(ConanFile):
         "fPIC": [True, False],
         "with_openssl": [True, False],
         "with_winssl": [True, False],
-        "with_darwinssl": [True, False],
-        "no_ssl": [True, False],
+        "ssl": ["openssl", "darwinssl", "winssl", AUTO_SSL, NO_SSL]
```

Comment:
Could you please stick with our usual, `with_` syntax?

Reply:
Will do!